src/indicator/algorithm/calculator/elderRay.js

```diff
@@ -0,0 +1,73 @@
+import d3 from 'd3';
+import exponentialMovingAverage from './exponentialMovingAverage';
+import {identity} from '../../../util/fn';
+import {rebind} from '../../../util/rebind';
+
+export default function() {
+
+    var value = identity;
+
+    var highValue = function(d, i) { return d.high; },
+        lowValue = function(d, i) { return d.low; };
+
+    var ema = exponentialMovingAverage()
+        .windowSize(13);
+
+    var elderRay = function(data) {
+
+        ema.value(value);
+        var x = ema(data);
+
+        var elderRay = d3.zip(data, x)
+            .map(function(d) {
```

Comment:
Yes I had a think about the leading undefined values and opted to leave them in, not knowing if there was a preference. I'll remove them though.